vuepress/docs/next/tutorials/devops/installation/google-cloud-platform/README.md

```diff
@@ -0,0 +1,249 @@
+It may take a few minutes for the `Address` to be populated.
+6. Verify you can access the web application by going to the `EXTERNAL-IP/hello` address, using the 
+`Address` from the previous step. You should see the following:
+```
+Hello, world!
+Version: 1.0.0
+Hostname: hello-app
+```
+
+Note the external IP address of your ingress controller since you’ll need it for the application configuration. 
+The Entando deployment exposes an environment variable to set the ingress controller to be used as part of the deployment. That variable is `ENTANDO_INGRESS_CLASS` and should be set to `nginx` in deployments to GKE (this is documented in the application instructions below as well)
+
+### Install the Entando Custom Resource Definitions (CRDs)
+Once per cluster you need to deploy the `Entando Custom Resources`. 
+
+1. Download and unpack the CRDs for the appropriate release: <https://github.com/entando-k8s/entando-k8s-custom-model/releases/tag/v6.1.5>
+   - Note: Any version later than 6.1.5 will work for deployment on GKE
+```
+curl -sfL https://github.com/entando-k8s/entando-k8s-custom-model/archive/v6.1.5.tar.gz | tar xvz
+```
+2. Change into the root of the downloaded directory: `cd entando-k8s-custom-model-6.1.5`
+3. Deploy the CRDs: `kubectl create -f src/main/resources/crd/`
+
+## Deploy Your Entando Application
+You can now deploy your Entando applications to GKE.
+
+### Setup and Deploy
+1. Download and unpack the entando-helm-quickstart release you want to use from here:
+<https://github.com/entando-k8s/entando-helm-quickstart/releases>
+   - e.g. `curl -sfL https://github.com/entando-k8s/entando-helm-quickstart/archive/v6.2.0-sprint4-rc.tar.gz | tar xvz`
+   - See the included README file for more information on the following steps.
+2. Edit `values.yaml`in the root directory: 
+   - Set `supportOpenshift: false`
+   - Set `ENTANDO_DEFAULT_ROUTING_SUFFIX` to the IP value of your `nginx` controller plus .nip.io
+      - For example: `ENTANDO_DEFAULT_ROUTING_SUFFIX: 35.223.161.214.nip.io`
+      - We’re using <https://nip.io> because we need wildcard dns address resolution however nip.io is not required. If your enterprise has a different internal dns resolution scheme for development instances you can use that or other alternative dns services like xip.io.
+   - If not already present, set these values to utilize nginx as the ingress controller and file system groups for persistent volume access:
+      - `ENTANDO_REQUIRES_FILESYSTEM_GROUP_OVERRIDE: "true"`
+      - `ENTANDO_INGRESS_CLASS: "nginx"`
+   - You can also disable the operator limit checking with this setting:
+      - `ENTANDO_K8S_OPERATOR_IMPOSE_DEFAULT_LIMITS: "false"`
+   - See [Appendix B](#appendix-b-example-values-yaml-file-for-helm-quickstart) for an example values.yaml 
+3. Create the Entando namespace: `kubectl create namespace entando`
+4. Update helm dependencies: `helm dependency update`
```

Comment:
@joewhite101 , @w-caffiero-entando , this uses Helm 3 which comes pre-installed for Google Cloud Shell. I didn't have any issues but is that a concern?

Reply:
Pretty sure the helm quickstart won't work with the helm 3 client

Reply:
@joewhite101 Entando started up fine and I ran through the steps with Google Cloud Shell twice. The command line arguments were slightly different but I documented what worked for me.  I just double checked the version and it's definitely Helm 3.
`n_shaw@cs-6000-devshell-vm-2b5c256d-2302-420e-8133-d10cc25b203c:~$ helm version
version.BuildInfo{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"}
n_shaw@cs-6000-devshell-vm-2b5c256d-2302-420e-8133-d10cc25b203c:~$ ls
entando-helm-quickstart-6.2.0-sprint4-rc  entando-k8s-custom-model-6.1.5  hello  ingress-resource.yaml  README-cloudshell.txt`

Reply:
Discussed with @joewhite101 , moving ahead with this version.